### ISSUE TITLE
Getting label names from collection schema

### ISSUE THREAD

**Issue by mazing** [aslagle/reactive-table#323]
Is it possible to omit the label option when specifying fields, and let it look for labels in my collection schema?

It seems redudant to write all the label names again.

**Comment by aslagle** [comment]
No, I haven't done anything like that. I'd rather avoid dependencies on other packages. Could you pull the labels out of your collection schema when you create the fields?
